### ISSUE TITLE
f-droid

### ISSUE THREAD

**Issue by obfusk** [obfusk/sokobang#11]
* [x] https://gitlab.com/fdroid/fdroiddata/-/merge_requests/7356
* [x] README
* [x] badges

NB: available in [obfusk's f-droid repo](https://obfusk.dev/fdroid/repo?fingerprint=2A21B7FFC93B878724B1991C05DAE113C72B93A556C193F49B5D3342884798B7).

**Comment by licaon-kter** [comment]
Fail to build?
https://f-droid.org/wiki/index.php?title=dev.obfusk.sokobang/lastbuild_1000203001&oldid=288789

**Comment by obfusk** [comment]
Thanks!

Yeah, the build metadata needs to be modified (it's missing the `ln -sf "$$python-for-android$$" .p4a` that `dev.obfusk.jiten` already has.) ...which I need to do anyway now that I figured out how to get not just reproducible builds between CI and my own build VM but also identical builds on the F-Droid buildserver.

I was half-expecting something to break but you noticed the failed build first :sweat_smile:

This update wasn't very important (the only significant difference is that it builds reproducibly), so I'll probably wait until I've tested it using a local buildserver.
